### PR TITLE
Use e2e.test binary instead of test/e2e in example invocations

### PR DESCRIPTION
**Pull request by wongma7**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:  ~the example invocation should use the binary instead of ginkgo because ideally the reader will only need to download the e2e.test binary and not need to clone the repo and/or have ginkgo in their PATH~
test/e2e assumes the user has cloned k/k and is working inside it, we want to document the case where the user has downloaded the binary and doesn't need to wait for anything to build

/sig storage

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by wongma7** [comment]
/cc @pohly 
/cc @msau42 
PTAL, thank you!!

File: test/e2e/storage/external/README.md [hunk 1]
```diff
@@ -32,18 +32,14 @@ prefix.
 To run just those tests for the example above, put that content into
 `/tmp/hostpath-testdriver.yaml` and invoke:
 
-    ginkgo -p -focus='External.Storage.*csi-hostpath' \
-           -skip='\[Feature:|\[Disruptive\]' \
-           ./test/e2e \
-           -- \
-           -storage.testdriver=/tmp/hostpath-testdriver.yaml
+    e2e.test -ginkgo.focus='External.Storage.*csi-hostpath' \
```
Comment (msau42) [inline test/e2e/storage/external/README.md]: One downside is that we lose the parallelism.  When I spoke with test-infra, they said that ginkgo -p flag will invoke the e2e binary multiple times with different -ginkgo.prallel.node values to manage the parallelism.

So I think we can just replace "./test/e2e" with the e2e binary in the original command

Comment (wongma7) [reply test/e2e/storage/external/README.md]: I see, done

Comment (pohly) [reply test/e2e/storage/external/README.md]: My original intent was to keep this README.md generic, i.e. not assume that this package is used only in the Kubernetes e2e.test binary. But I guess anyone who does that is able to figure out how to generate an `e2e.test` binary or invoke `ginkgo` on the source, so this change is fine.

**Pull request merged**